sql/core/src/test/scala/org/apache/spark/sql/CollationExpressionWalkerSuite.scala

```diff
@@ -607,6 +607,8 @@ class CollationExpressionWalkerSuite extends SparkFunSuite with SharedSparkSessi
       // need to skip as plans differ in STRING <-> STRING COLLATE UTF8_LCASE
       "current_timezone",
       "schema_of_variant",
+      "hash",
+      "xxhash64",
```

Comment:
nit: Is this the right group for these expressions? Don't we expect results to differ, as hashes are differnet?